pinot-controller/src/main/java/com/linkedin/pinot/controller/ControllerConf.java

```diff
@@ -41,6 +41,7 @@
   private static final String RETENTION_MANAGER_FREQUENCY_IN_SECONDS = "controller.retention.frequencyInSeconds";
   private static final String VALIDATION_MANAGER_FREQUENCY_IN_SECONDS = "controller.validation.frequencyInSeconds";
   private static final String STATUS_CHECKER_FREQUENCY_IN_SECONDS = "controller.statuschecker.frequencyInSeconds";
+  private static final String REALTIME_SEGMENTS_RELOCATION_MANAGER_FREQUENCY_IN_MINUTES = "controller.segment.relocation.frequencyInMinutes";
```

Comment:
You can change this to controller.segment.relocation.frequency and accept strings using PeriodFormatter. Also, you may want to consider controller.realtime.segment.relocation.frequency